tap_shopify/streams/graphql/gql_base.py

```diff
@@ -100,9 +101,9 @@ def call_api(self, query_params):
         except ShopifyAPIError as gql_error:
             LOGGER.error("GraphQL Error %s", gql_error)
             raise ShopifyAPIError("An error occurred with the GraphQL API.") from gql_error
-        except Exception as e:
+        except Exception as exception:
             LOGGER.error("Unexpected error occurred.",)
-            raise e
+            raise exception
```

Comment:
This is not a necessary change but to make the error handling consistant across the codebase change the variable exception to exc as it is used everywhere in the tap or change all occurrences of exc to exception